proto/mixer.proto

```diff
@@ -633,6 +631,62 @@ message GetStatSeriesResponse {
   map<string, double> series = 1;
 }
 
+// Request for GetStatAll service.
+message GetStatAllRequest {
+  // dcids of the place.
+  repeated string places = 1;
+  // dcids of the stat var.
+  repeated string stat_vars = 2;
+}
+
+// Response for GetStatAll service.
+// The response is a two level map, with the first level keyed by place dcid,
+// and the second level keyed by the stat var dcid.
+// Each leaf object contains multiple source series with <date, value> object
+// and observation metadata.
+//
```

Comment:
maybe it's worth converting these to camelcase as well, and noting here that the conversion happens somewhere..

Reply:
done